README.md

```diff
@@ -56,48 +73,79 @@ $ source setup/activate_serve.sh
+## 3. Updating the e-mission-\* plugins or adding new plugins
 
-1. installing a local server,
-2. running it, 
-3. loading it with test data, and
-4. running analysis on it
+[![osx-build-ios](https://github.com/e-mission/e-mission-phone/actions/workflows/ios-build.yml/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/ios-build.yml)
+[![osx-build-android](https://github.com/e-mission/e-mission-phone/actions/workflows/android-build.yml/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/android-build.yml)
 
-are available in the [e-mission-server README](https://github.com/e-mission/e-mission-server/blob/master/README.md).
+__Important__
 
-In order to make end to end testing easy, if the local server is started on a HTTP (versus HTTPS port), it is in development mode.  By default, the phone app connects to the local server (localhost on iOS, [10.0.2.2 on android](https://stackoverflow.com/questions/5806220/how-to-connect-to-my-http-localhost-web-server-from-android-emulator-in-eclips)) with the `prompted-auth` authentication method. To connect to a different server, or to use a different authentication method, you need to create a `www/json/connectionConfig.json` file. More details on configuring authentication [can be found in the docs](https://github.com/e-mission/e-mission-docs/blob/master/docs/install/configuring_authentication.md).
+Most of the recent issues encountered have been due to incompatible setup. We
+have now:
+- locked down the dependencies,
+- created setup and teardown scripts to setup self-contained environments with
+  those dependencies, and
+- CI enabled to validate that they continue work.
 
-One advantage of using `skip` authentication in development mode is that any user email can be entered without a password. Developers can use one of the emails that they loaded test data for in step (3) above. So if the test data loaded was with `-u shankari@eecs.berkeley.edu`, then the login email for the phone app would also be `shankari@eecs.berkeley.edu`.
+If you have setup failures, please compare the configuration in the **passing CI
+builds** with your configuration. That is almost certainly the source of the error.
 
-Updating the e-mission-\* plugins or adding new plugins
----
-[![osx-build-ios](https://github.com/e-mission/e-mission-phone/actions/workflows/ios-build.yml/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/ios-build.yml)
-[![osx-build-android](https://github.com/e-mission/e-mission-phone/actions/workflows/android-build.yml/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/android-build.yml)
+### Tested on
+__MacOS__
+- Intel chip, MacOS Ventura 13.6
+- Intel chip, MacOS Ventura 13.5.2
+- Intel chip, MacOS Ventura 13.0
+- Intel chip, MacOS Monterey 12.6.7
```

Comment:
I am really reluctant to put versions into the README because it is not clear who is taking on the responsibility of keeping this updated. Whose laptops are these? How often do they build the app? When is the last time anybody with that laptop built this app? "Tested on" doesn't mean anything unless we are able to actually fill in those results.